spk/nzbget/src/nzbget.conf

```diff
@@ -85,8 +85,14 @@ ScriptDir=/usr/local/nzbget/share/nzbget/scripts
 
```

Comment:
File has changed permissions. Any particular reason? Otherwise best to revert that.